tests/util/test_traceutils.py

```diff
@@ -0,0 +1,65 @@
+from unittest.mock import patch
+import pytest
+
+from instana.tracer import InstanaTracer
+from instana.util.traceutils import (
+    extract_custom_headers,
+    get_active_tracer,
+    get_tracer_tuple,
+    tracing_is_off,
+)
+from instana.singletons import agent, tracer
+
+
+class TestTraceUtils:
+    @pytest.fixture(autouse=True)
+    def _resource(self):
+        pass
+
```

Comment:
No need to have this if you are not setting up and tearing down the test execution.